src/Catrobat/Controller/Ci/BuildApkController.php

```diff
@@ -23,13 +23,12 @@ class BuildApkController extends Controller
    * @Route("/ci/build/{id}", name="ci_build", defaults={"_format": "json"},
    *   requirements={"id": "\d+"}, methods={"GET"})
    *
-   * @param Request $request
    * @param Program $program
    *
    * @return JsonResponse
    * @throws \Exception
    */
-  public function createApkAction(Request $request, Program $program)
```

Comment:
Why did you remove that?

Reply:
'cause it was unused... 
(just produces a warning)